packages/management/addons-manager-v2/bundle/config/values.yaml

```diff
@@ -13,3 +13,4 @@ tanzuAddonsManager:
     healthzPort: 18316
   featureGates:
     clusterBootstrapController: false
+    packageInstallStatus: false
```

Comment:
It seems like we need to duplicate the values at multiple levels.

https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/management/framework/bundle/config/values.yaml#L28

https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/management/tkg/bundle/config/values.yaml#L39